code/game/objects/items/stacks/hot_ice.dm

```diff
@@ -0,0 +1,26 @@
+/obj/item/stack/hot_ice
+	name = "hot ice"
+	icon_state = "hot-ice"
+	item_state = "hot-ice"
+	singular_name = "hot ice"
+	icon = 'icons/obj/stack_objects.dmi'
+	custom_materials = list(/datum/material/hot_ice=10000)
```

Comment:
why is the stack 10k materials thats not how stacks work

Reply:
if the system supports it, whats wrong with it

Reply:
uh is that value because i have no idea why it shouldnt be there, i basically copied the rods one and modified some stuff in it

Reply:
@iamgoofball ah yes thank you goofball for another useless comment. The system also allows you to add spells with 100 brute and 0 cool down.

Anyways, we have a standardized sheet material amount which is defined somewhere, check other sheets like gold to see what the define is its like MINERAL_MATERIAL_AMOUNT

Reply:
i'll set it to half the amount of MINERAL_MATERIAL_AMOUNT, so 1000, in the next commit

Reply:
No leave it at the full value of the define, sheets are the center of balance so if you make them 1k you could dupe materials by making tiles worth 500 etc

Reply:
alright